go.mod

```diff
@@ -37,7 +37,6 @@ require (
 	github.com/gobuffalo/flect v1.0.2 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
-	github.com/golang/mock v1.6.0 // indirect
```

Comment:
is this `mock` removal on purpose?

Reply:
Yes. The correct module is `go.uber.org/mock v0.4.0`.